Modules/_ctypes/libffi_osx/powerpc/ppc-ffi_darwin.c

```diff
@@ -650,7 +650,7 @@ ffi_call(
 		  +---------------------------------------+ 160
 		  | result area 8                         |
 		  +---------------------------------------+ 168
-		  | alignement to the next multiple of 16 |
```

Comment:
Same comment with this file:

I realize we have our own copy of libffi for OS-X. Wouldn't changes to these files cause porting upstream changes more difficult? I suggest we not change any libffi files. See Modules/_ctypes/libffi_osx/README.pyobjc.

Reply:
reverted.